edc-extensions/business-partner-validation/src/main/java/org/eclipse/tractusx/edc/validation/businesspartner/BusinessPartnerValidationExtension.java

```diff
@@ -56,7 +55,7 @@ public class BusinessPartnerValidationExtension implements ServiceExtension {
      * </pre>
      */
     // TODO replace with TX namespace
-    public static final String BUSINESS_PARTNER_CONSTRAINT_KEY = EDC_NAMESPACE + "BusinessPartnerNumber";
+    public static final String BUSINESS_PARTNER_CONSTRAINT_KEY = "BusinessPartnerNumber";
```

Comment:
there already is a constant `CoreConstants.TX_NAMESPACE`

Reply:
Yeah it has been removed here since it's not needed

Reply:
wanna remove the TODO in L47 then?